pyproject.toml

```diff
@@ -124,7 +124,7 @@ broadcaster = {extras = ["redis"], version = "^0.2.0"}
 sqlalchemy-citext = "^1.8.0"
 coolname = "^1.1.0"
 python-slugify = "^5.0.2"
-cryptography = "3.3.2" # Downgraded to avoid alpine build error
```

Comment:
@mariobehling it was downgraded intensionally, so I don't know it should be updated or not.